packages/libxshmfence.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Libxshmfence < Package
+  description 'libxshmfence: A library that exposes a event API on top of Linux futexes'
+  homepage 'http://t2sde.org/packages/libxshmfence.html'
+  version '1.32'
+  source_url 'http://xorg.freedesktop.org/releases/individual/lib/libxshmfence-1.2.tar.bz2'
```

Comment:
Can you please change the `http:` to `https:`?  This will make the url more secure.